content/actions/reference/workflow-syntax-for-github-actions.md

```diff
@@ -876,6 +876,37 @@ strategy:
 
 {% endnote %}
 
+##### Using environment variables in a matrix
```

Comment:
This is good and this will work. Consider the simpler, more straight forward form:

```yaml
name: Node.js CI
on: [push]
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
       include:
         - node-version: 10.x
           site: "prod"
           datacenter: "site-a"
         - node-version: 12.x
           site: "dev"
           datacenter: "site-b"
    steps:
    - name: Echo site details
      env:
        SITE: ${{ matrix.site }}
        DATACENTER: ${{ matrix.datacenter }}
      run: echo $SITE $DATACENTER
```

Reply:
Thanks so much for the review @ericsciple! I've updated the example now 👍